### ISSUE TITLE
Blank white screen in the OpenDeck GUI using an Nvidia GPU

### ISSUE THREAD

**Issue by Danie10** [ninjadev64/OpenDeck#25]
I did a new installation of the git package from AUR because the non-git version did not build. I already have the udev rules setup as I've been using other Streamdeck software. I did also try reloading the udev rules, and I unplugged my second Streamdeck.

When I start this app up, I just get a blank grey window. There is no log file created yet I see.

If I start it from the command line this is the output:

```
[2024-10-03][21:18:45][tiny_http][DEBUG] Server listening on 0.0.0.0:57118
[2024-10-03][21:18:45][tiny_http][DEBUG] Running accept thread

(opendeck:506117): IBUS-WARNING **: 23:18:45.391: Unable to connect to ibus: Could not connect: Connection refused
[2024-10-03][21:18:45][opendeck::events][DEBUG] Registered plugin com.amansprojects.starterpack.sdPlugin
AcceleratedSurfaceDMABuf was unable to construct a complete framebuffer
AcceleratedSurfaceDMABuf was unable to construct a complete framebuffer
AcceleratedSurfaceDMABuf was unable to construct a complete framebuffer
AcceleratedSurfaceDMABuf was unable to construct a complete framebuffer
AcceleratedSurfaceDMABuf was unable to construct a complete framebuffer
AcceleratedSurfaceDMABuf was unable to construct a complete framebuffer
AcceleratedSurfaceDMABuf was unable to construct a complete framebuffer
AcceleratedSurfaceDMABuf was unable to construct a complete framebuffer
AcceleratedSurfaceDMABuf was unable to construct a complete framebuffer
AcceleratedSurfaceDMABuf was unable to construct a complete framebuffer
AcceleratedSurfaceDMABuf was unable to construct a complete framebuffer
AcceleratedSurfaceDMABuf was unable to construct a complete framebuffer
```

**Comment by ninjadev64** [comment]
Do you have an Nvidia GPU? Please try using the WEBKIT_DISABLE_DMABUF_RENDERER environment variable (`WEBKIT_DISABLE_DMABUF_RENDERER=1 opendeck`).

**Comment by Danie10** [comment]
Yes I do have Nvidia. Ok thanks Ill try that and report back.

**Comment by Danie10** [comment]
Great thanks that sorted out the problem. App is showing buttons fine, but although my Stream Decks are recognised and their button presses show on the app screen, there is no display on the deck's buttons. Brightness is set to max.

BTW yes impressive selection of plugins - would be nice tho to have a popup or something giving some help about what they are a nd what functions they have, before installing.

**Comment by ninjadev64** [comment]
> there is no display on the deck's buttons.

Someone else had the same issue today, I'll let you know when there's a fix.

**Comment by ninjadev64** [comment]
@Danie10 I've got a Stream Deck Mini of my own now, and the issue where the deck's buttons are blank is fixed in v2.1.0.
